tests/console/check_system_info.pm

```diff
@@ -27,6 +27,8 @@ use version_utils;
 use registration;
 use List::MoreUtils 'uniq';
 
+my $flag;
+
```

Comment:
i suggest new sub function such as set_productor, set_addon can be created and be called within check_xxx function.